scripts/demos/common/amaru-node.sh

```
@@ -0,0 +1,102 @@
+    dev) echo "debug" ;;
+    release) echo "release" ;;
+    *) echo "$BUILD_PROFILE" ;;
+  esac
+}
+
+amaru_node_binary() {
+  echo "${CARGO_TARGET_DIR:-$AMARU_DIR/target}/$(target_profile_dir)/amaru"
+}
+
+build_amaru_node_binary() {
+  cd "$AMARU_DIR"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)